src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -1205,13 +2099,135 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
             }
         }
 
+        private static async Task SeedDatabase(
+            string contextId,
+            Release release,
+            IReadOnlyList<Subject>? subjects = null,
+            IReadOnlyList<ReleaseSubject>? releaseSubjects = null,
+            IReadOnlyList<Footnote>? footnotes = null)
+        {
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
+            {
+                await statisticsDbContext.Release.AddAsync(release);
+
+                if (!subjects.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.Subject.AddRangeAsync(subjects!);
+                }
+
+                if (!releaseSubjects.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.ReleaseSubject.AddRangeAsync(releaseSubjects!);
+                }
+
+                if (!footnotes.IsNullOrEmpty())
+                {
+                    await statisticsDbContext.Footnote.AddRangeAsync(footnotes!);
+                }
+
+                await statisticsDbContext.SaveChangesAsync();
+
+                await contentDbContext.AddAsync(new Content.Model.Release
+                {
+                    Id = release.Id
+                });
+                await contentDbContext.SaveChangesAsync();
+            }
+        }
+
+        private static async Task<Either<ActionResult, Footnote>> CreateFootnoteWithConfiguration(
+            Guid releaseId,
+            string contextId,
+            string? content = null,
```

Comment:
Could use `string? content = ""` here to avoid `content ??= string.Empty` below.

Reply:
done :)

Reply:
Cool, my fault though, it should have been `string content = ""` rather than `string? content = ""` 😆

Reply:
done :)